models/intermediate/int_recharge__customer_details.sql

```diff
@@ -38,7 +38,7 @@ with customers as (
 ), subscriptions as (
     select 
         customer_id,
-        count(subscription_id) as calculated_active_subscriptions -- this value will differ from the recharge-provided number_active_subscriptions until we get confirmation from recharge how this is calculated on their end
+        count(subscription_id) as calculated_subscriptions_active_count -- this value will differ from the recharge-provided subscriptions_active_count until we get confirmation from recharge how this is calculated on their end
```

Comment:
Should we still include this comment? What ever came of this confirmation?

Reply:
@fivetran-joemarkiewicz good point. We did not get confirmation back, [but the updated api docs better explain what it should be (the old docs did not).](https://developer.rechargepayments.com/2021-11/customers/customers_object) Our calculation generally matches up. 

That being said, I did some more digging and found a pattern when the count we calculate and the value provided by recharge do not match. A mismatch occurs for subscription records that were created on a particular day (I'll can send the query internally if you're interested) but not updated since. I'm wondering if, for example, an issue occurred on that day, but the counts don't get updated unless the counter is triggered by something like an update. 

Therefore I do think this calculated count is more accurate, but it also isn't used in any downstream models. I made a comment in the recharge.yml and also updated the comment in the code, but wondering if this warrants an entry into the decision log?

Reply:
Thanks for the investigation here @fivetran-catfritz! Based off this I feel comfortable including our calculation and the additional documentation you provided is great to keep users clued into what is occurring under the hood.